recipe/build.sh

```diff
@@ -115,10 +119,10 @@ cd build
   --disable-bootstrap \
   --disable-multilib \
   --enable-long-long \
-  --enable-default-pie \
```

Comment:
Why remove this?

Reply:
Only removed a duplicate argument.  It's still included earlier elsewhere in the configure cmdline.